diff --git a/models_mae.py b/models_mae.py
index 880e28f..5bd790f 100644
--- a/models_mae.py
+++ b/models_mae.py
@@ -101,9 +101,9 @@ class MaskedAutoencoderViT(nn.Module):
         assert imgs.shape[2] == imgs.shape[3] and imgs.shape[2] % p == 0

         h = w = imgs.shape[2] // p
-        x = imgs.reshape(shape=(imgs.shape[0], 3, h, p, w, p))
+        x = imgs.reshape(imgs.shape[0], 3, h, p, w, p)
         x = torch.einsum('nchpwq->nhwpqc', x)
-        x = x.reshape(shape=(imgs.shape[0], h * w, p**2 * 3))
+        x = x.reshape(imgs.shape[0], h * w, p**2 * 3)
         return x

     def unpatchify(self, x):
